### ISSUE TITLE
Refactor: Admin page

### ISSUE THREAD

**Issue by dallonf** [GameMakersToolkit/team-finder#35]
* The data flow could be greatly simplified by leveraging React Query - right now it's making HTTP requests directly.
* It's using state to manage tabs where routing would be more appropriate
* Names are a bit silly and hard to parse (""isAngryAdmin", "lol nope", "whoops, my bad", etc.)

**Comment by Willdotwhite** [comment]
Closed as Not Needed - rebuilding the UI from scratch removes the need for this refactor.
